### ISSUE TITLE
Nested Output Modifiers, issue introduced in 2.4.3

### ISSUE THREAD

**Issue by sarahetter** [modxcms/revolution#12943]
### Summary
When using nested output modifiers, the inside template tag isn't rendering properly.

### Step to reproduce
I have a modifier based on a TV:
`[[*header_features:is=``:then=`[[$home_site1]]`]]

The contents of [[$home_site1]] are:
```
<section class="site1" style="background-image: url([[*home_site1_background]])">
    <div class="centered">
        [[*home_site_phone_img:is=``:then=``:else=`<img src="[[*home_site_phone_img]]">`]]
        <article class="stand-out site-caption">
            <h1>[[*home_site1_greenbox_title]]</h1>
            [[*home_site1_greenbox_content]]
            [[*home_site1_greenbox_button_url:is=``:then=``:else=`
                <a href="[[*home_site1_greenbox_button_url]]" class="button">[[*home_site1_greenbox_button_text]]</a>
            `]]
        </article>
        <div class="goal-statement">[[*home_site1_tagline]]</div>
    </div>
</section>
```

### Observed behavior
The html output by this is:
```
<img src="">`]]
        <article class="stand-out site-caption">
            <h1>Title Here</h1>
            <p>Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrumsed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus sagittis augue laoreet rutrumsed odio dui lacus vel.</p>
            
                <a href="http://google.com" class="button">Click here</a>
            
        </article>
        <div class="goal-statement">Lorem Ipsum</div>
    </div>
</section>
```

### Expected behavior
The output should be the contents of [[$home_site1]]. However the first few lines are missing, and there is a non-parsed **`]]** in there. If I change the contents of [[$home_site1]] to just "Foobar", there is no output at all.

**Comment by pixelchutes** [comment]
I believe this was fixed today by #12939, slated for `2.4.4`

**Comment by opengeek** [comment]
Yes, this is a duplicate of #12906
